### ISSUE TITLE
could not be compiled

### ISSUE THREAD

**Issue by jpalmer8924** [Looking-Glass/UnrealPlugin_Issues#2]
Hello i am getting the MyProject could not be compiled. try rebuilding from source manually. I then installed visual basic 2019 and then also installed the C++ add on, this made the process look like it was going to work even add a binaries directory with two .dlls but then still ended off with the same error message. I am not to familiar with c++ or visual basic but the buttons inside visual basic like bake for example does not seem to do anything

**Comment by Batname** [comment]
@jpalmer8924 Hello, I try to reproduce it with VS 2019 I'll let you know about it today. If I can't reproduce it  we will setup chat or call with you

**Comment by Batname** [comment]
![1](https://user-images.githubusercontent.com/6189196/60507414-6695db00-9cc8-11e9-8566-10737026af87.JPG)
 @jpalmer8924  I can't reproduce this issue with 4.22 version and Visual Studio 2019 

1. Could you try to build this version 
[UnrealSDK-1.1.8.zip](https://github.com/Looking-Glass/UnrealPlugin_Issues/files/3349676/UnrealSDK-1.1.8.zip)
2. If you still get an error. Let set up a call or chat here is my phone +31 6 491 66 836

**Comment by daraburke** [comment]
Hi, I couldn't get version 1.1.0 to build with 2019. I switched over to another machine with 4.22 and VS 2017 on it and it worked perfectly.

**Comment by Batname** [comment]
@daraburke  If it possible could you send the Unreal Engine Logs from Project/Saved folder. It will be useful to understand the reason.

**Comment by daraburke** [comment]
@Batname I will have a look and see if I can find the relevant project logs. If not I will try to reproduce it here again.

**Comment by daraburke** [comment]
Hi @Batname 

Sorry for the delay, I couldn't find the original log files, I think I deleted those test projects. Here is a log file from today where it failed to build and the asked to build from source. This machine has 4.22.2 and VS 2019 installed. Maybe I am missing some libraries on VS 2019 on my end. This was using the first version of the plugin that was released, not the current version.

[LookingGlassTest.log](https://github.com/Looking-Glass/UnrealPlugin_Issues/files/3406515/LookingGlassTest.log)

**Comment by reidbuz** [comment]
I have not been able to successfully compile version 1.21 with my own Looking Glass UnReal_projects, so here's what I did...

1) Open the UnRealSDK.uproject inside the UnrealSDK-1.2.1 folder using UnRealEditor 4.22.3 and let it build the plugins. This succeeded, and the UnRealSDK project opened perfectly.
2) Quit UnRealEditor 4.22.3
2) Copied the Plugins folder from inside the UnrealSDK-1.2.1 folder, and pasted it inside my own project folder.
3) Opened UnRealEditor 4.22.3, then selected my own project from the list of projects, and opened it.
4) It opened and the whole set of Looking Glass plugins appeared and work perfectly inside my project.

I offer this as a work around if you have no success building the UnReal_Looking Glass plugins in your own UnReal_project like I did.
